packages/dds/tree/src/shared-tree/schematizingTreeView.ts

```diff
@@ -173,19 +173,20 @@ export class SchematizingSimpleTreeView<
 		}
 
 		this.runSchemaEdit(() => {
+			const schema = this.viewSchema.viewSchemaAsStored;
 			const mapTree = mapTreeFromNodeData(
 				content as InsertableContent | undefined,
 				this.rootFieldSchema,
 				this.nodeKeyManager,
 				{
-					schema: this.checkout.storedSchema,
```

Comment:
This provided the schema from before the change (which is empty) not the new schema when validating.